pynucastro/rates/rate.py

```diff
@@ -490,12 +491,13 @@ def _update_label_properties(self):
         assert isinstance(self.labelprops, str)
         try:
             assert len(self.labelprops) == 6
-        except:
+        except AssertionError:
```

Comment:
This would be cleaner as `if len(self.labelprops) != 6:`

Reply:
fixed